### ISSUE TITLE
WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)

### ISSUE THREAD

**Issue by federicoaposada** [federicoaposada/CUNA#2]
#### Issue
`Ensures every HTML document has a lang attribute` ([html-has-lang](https://accessibilityinsights.io/info-examples/web/html-has-lang))
#### Target application
[DatosCUIL](http://servicioswww.anses.gob.ar/ooss2/)
#### Element path
html
#### Snippet
`<html xmlns="http://www.w3.org/1999/xhtml">`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Chrome version 103.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
